pipelines/WTRCSP.py

```diff
@@ -0,0 +1,17 @@
+from pyriemann.estimation import Covariances
```

Comment:
can the solver take both yml and py files?

Reply:
Yes =)

Reply:
Good idea!